Sources/Purchasing/Purchases/PurchasesAreCompletedBy.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  PurchasesAreCompletedBy.swift
+//
+//  Created by James Borthwick on 2024-05-30.
+
+import Foundation
+
+/// Where responsibility for completing purchase transactions lies.
+@objc(RCPurchasesAreCompletedBy)
+public enum PurchasesAreCompletedBy: Int {
```

Comment:
That's a weird name for an `enum` 😇

Reply:
LOL it is, we have a many-message Slack thread about it!